solutions/standard/main.tf

```diff
@@ -0,0 +1,39 @@
+########################################################################################################################
+# Resource Group
+########################################################################################################################
+
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.4"
+  resource_group_name          = var.existing_resource_group == false ? var.resource_group_name : null
+  existing_resource_group_name = var.existing_resource_group == true ? var.resource_group_name : null
+}
+
+########################################################################################################################
+# Event Notifications
+########################################################################################################################
+
+locals {
+  service_credential_names = var.service_credential_names
+  # KMS Add on
+  existing_kms_instance_crn = var.existing_kms_instance_crn != null ? var.existing_kms_instance_crn : null
+  kms_endpoint_url          = var.kms_encryption_enabled ? var.kms_endpoint_url != null ? var.kms_endpoint_url : format("https://%s.kms.cloud.ibm.com", var.region) : null
+  kms_root_key_id           = var.kms_root_key_id != null ? var.kms_root_key_id : null
+}
+
```

Comment:
Add code to create the KMS key if one is not passed in (see SCC DA for reference)